service/static/js/rest_api.js

```diff
@@ -393,5 +435,176 @@ $(function () {
         });
 
     });
+
+
+// ************************************************************************************************************************
+// Address Functions
+// ************************************************************************************************************************
+
+    // ****************************************
+    // Create an Address
+    // ****************************************
+    $("#addr-create-btn").click(function () {
```

Comment:
When I create an address using the UI, the customer's information is cleared out of the customer part of the form. Not sure if we want to show the customer information after an address is added

Reply:
> When I create an address using the UI, the customer's information is cleared out of the customer part of the form. Not sure if we want to show the customer information after an address is added

You mean we should let the customer's information stay? ok